conans/model/info.py

```diff
@@ -168,7 +168,7 @@ def copy(self):
         return RequirementsInfo(data)
 
     def serialize(self):
-        return [str(r) for r in sorted(self._data.values())]
+        return [r.dumps() for r in self._data.values()]
```

Comment:
Any specifics for removing this sort? It does not look necessary at first glance, the important one is the dumps one, but curious to know the reasoning behind removing this one :)

Reply:
Yes, it was crashing! Because the value type doesn't implement less-than.

The problem is that this code was not used/covered at all.